src/app/page/UserProfile/UserOverview.jsx

```diff
@@ -29,8 +33,33 @@ const UserStatus = ({ profile, setProfile, view }) => {
     setProfile(_.cloneDeep(profile));
   }
 
+  const handleAddressSelect = () => {
+    const addrObj = autocomplete.getPlace();
+    const addr = addrObj.formatted_address;
+
+    if (addr) {
+      profile["address"] = addr;
+      setProfile(_.cloneDeep(profile));
+    }
+  };
+
+  const handleScriptLoad = () => {
+    const options = {
+      type: ["(cities)"],
+    };
+    const addrElem = document.getElementById("address");
+
+    autocomplete = new google.maps.places.Autocomplete(addrElem, options);
```

Comment:
Can we make it clear that this is coming from the `window` object? Also should check that this script has loaded correctly. I believe `onError` is a required prop for react-load-script module you're using here.

Something to look at later (maybe i'll make a ticket for it) is we could re purpose the [usePaypal](https://github.com/factn/resilience-app/blob/master/src/app/hooks/usePaypal.js) hook to be a general `useScript` hook for situations like this. I think hooks are better for this async behavior than having it in a react element

Reply:
I can refactor it to use hooks for this situation and maybe generalize it for any scripts.

Reply:
Hooks would be nice!

Reply:
There is not a **useScript** hook for generic script loading.